### PR TITLE
Avoid unicode filename

### PR DESCRIPTION
**Pull request by tbg**
This is similar to 53e9d3ee100e0cf54d18eeae09dda6e09a29d090.

Over at https://github.com/cockroachdb/cockroach, we use bazel and it
chokes on this file:

```
java.io.IOException: Error extracting
[...]/com_github_maruel_panicparse_v2-v2.2.1.zip to [...]:
[...]/com_github_maruel_panicparse_v2/cmd/panic/internal/utf8/?tf8.go
(Illegal byte sequence)
```

which is why we're on a fork so far. It would be nice to avoid this.

There is an (eternal, it seems) [upstream discussion], so there is
little home of bazel starting to support non-unicode filenames.

I wonder if the loss in test coverage (if any) is acceptable in return
for avoiding build problems like the ones we've been seeing.

[upstream discussion]: https://github.com/bazelbuild/bazel/issues/374

### CODE REVIEW

**Comment by maruel** [comment]
Test failures are legit. Also there's probably 2~3 comments to update.

**Comment by tbg** [comment]
Thanks! Tests should be passing now. I looked for comments that needed changes (`git grep -i utf` essentially) and didn't find anything obvious. But I did add a comment in `utf8.go` explaining why it's not also a unicode filename.

File: cmd/panic/internal/utf8/utf8.go [hunk 1]
```diff
@@ -3,6 +3,8 @@
 // that can be found in the LICENSE file.
 
 // Package ùtf8 tests a package and function with non-ASCII names.
+// The filename itself does not contain unicode as this causes issues
```
Comment (maruel) [inline cmd/panic/internal/utf8/utf8.go:6]: Please insert an empty line "//" so this is in a separate paragraph. It makes godoc output much nicer.

Comment (tbg) [reply cmd/panic/internal/utf8/utf8.go:7]: Done.

**Comment by maruel** [comment]
Thanks!

**Pull request merged**